e2e/updatecli.d/success.d/golang/module.yaml

```diff
@@ -0,0 +1,19 @@
+name: "Test gomod"
```

Comment:
Module?

Reply:
Thanks I am reviewing the  title of those e2e tests